### PR TITLE
Update types

### PR DESCRIPTION
**Pull request by alchemistgo87**

### CODE REVIEW

File: src/listings.ts [hunk 1]
```diff
@@ -21,52 +21,52 @@ const {
   createPrintPurchaseReceiptInstruction,
   createCancelInstruction,
   createCancelListingReceiptInstruction,
-} = instructions;
+} = instructions
 
 export interface PostListingParams {
-  amount: number;
-  nft: Nft;
+  amount: number
```
Comment (austinmilt) [inline src/listings.ts:27]: why are you ditching the semicolons?

**Review by austinmilt — APPROVED** [review]
looks fine, but I'd prefer keepings semicolon puncutation

File: src/listings.ts (hunk 1 above)
Comment (alchemistgo87) [reply src/listings.ts:27]: Hey @austinmilt, I was actually using the .prettierrc from marketplace repo. Seems holaplex and marketplace repo follow slightly different conventions. I can update it to match it with that of holaplex, if you want me to.

File: src/offers.ts [hunk 2]
```diff
@@ -140,44 +140,44 @@ export class OffersClient extends Client {
...
-    const treasuryMint = new PublicKey(ah.treasuryMint);
-    const tokenAccount = new PublicKey(nft.owner.associatedTokenAccountAddress);
+    const ah = this.auctionHouse
+    const { publicKey, signTransaction } = this.wallet
+    const connection = this.connection
+    const auctionHouse = new PublicKey(ah.address)
+    const authority = new PublicKey(ah.authority)
+    const auctionHouseFeeAccount = new PublicKey(ah.auctionHouseFeeAccount)
+    const tokenMint = new PublicKey(nft.mintAddress)
+    const receipt = new PublicKey(offer.id)
```
Comment (kespinola) [inline src/offers.ts:168]: This needs to change to a pda lookup.

**Review by kespinola — CHANGES_REQUESTED** [review]
One spot where a receipt is referencing id instead of using pda lookup for the address.

**Review by kespinola — APPROVED** [review]

**Pull request merged**